dialog/dialog.css

```diff
@@ -158,4 +159,18 @@ dialog {
       }
     }
   }
+}
+
+@keyframes backdrop-blur {
+  0%   {
+    backdrop-filter: blur(5px);
+  }
+  
+  50%  {
+    backdrop-filter: blur(15px)
+  }
+
+  100% {
+    backdrop-filter: blur(25px)
+  }
 }
```

Comment:
Sorry, because of my lack of experience, I couldn't map an optimal way. I can't determine the best action I can take here. so I took two different approaches

- [First approach](https://github.com/argyleink/gui-challenges/pull/113/commits/0529bd75e407ae5b6420e442c4a434d4f3b92b37): I tried to add opacity to the keyframe itself resulted in better performance but still feels buggy.
-  [Second approach](https://github.com/argyleink/gui-challenges/pull/113/commits/1cc4db076202586d4847b2fe4df1826f5ed8481f): I removed the keyframe and opt to animate the opacity. This one feels better, but I may be missing something. 

This was fun to do. if you need me to work on it more, I don't mind.

Reply:
well don't feel so bad! I just played around for 45 minutes to try and get it too. I closed the feature/bug because it would take too much javascript and natural circumvention to do, for something so little. I think we could get away with a less ambitious backdrop animation, or some element restructuring too, but yeah, it's a lot.

i really appreciate you kick starting this work tho and trying some things out! ❤️

Reply:
> well don't feel so bad! I just played around for 45 minutes to try and get it too. I closed the feature/bug because it would take too much javascript and natural circumvention to do, for something so little. I think we could get away with a less ambitious backdrop animation, or some element restructuring too

Thanks, for the positive feedback. Although the pull request is closed, I did learn from this. I am happy with what I learned.

> i really appreciate you kick starting this work tho and trying some things out! ❤️

anytime, this was fun to do. Have a good day. ♥